src/lib/containers/RssFeed.tsx

```diff
@@ -34,16 +36,28 @@ export default class RssFeed extends React.Component<RssFeedProps, RssState> {
       itemId2MoreItem: {},
     }
   }
-
+  
   componentDidMount() {
+    this._isMounted = true
```

Comment:
I would suggest removing the _isMounted variable, if it's set to true at the top of this method then in theory it should always be true when being checked on lines 47 and 52.